transport/tchannel/outbound.go

```diff
@@ -198,8 +200,9 @@ func callWithPeer(ctx context.Context, req *transport.Request, peer *tchannel.Pe
 	deleteReservedHeaders(headers)
 
 	resp := &transport.Response{
-		Headers:          headers,
-		Body:             readCloser{bytes.NewReader(buf.Bytes())},
+		Headers: headers,
+		//Body:             readCloser{bytes.NewReader(buf.Bytes())},
+		Body:             buf,
```

Comment:
Thrift encoder does not close the buffer and as it this does not implement io.ReaderAt, encoder will create a duplicate copy from this buffer

Reply:
There are 2 thrift encoder implementations:
 - thrift/outbound.go (old)
 - thrift/outbound_nowire.go (new)

Both implementations close the response body.

I also see that the buffer is duplicated only by the older encoder implementation. I've addressed it by returning a response body that also implements io.ReaderAt. 

Is it safe to get rid of the older thrift/outbound.go encoder implementation?